draft-iab-privacy-partitioning.md

```diff
@@ -702,27 +710,31 @@ enough parties to recover identities.
 
 ## Violations by Insufficient Partitioning
 
-It is possible to define contexts that contain more than one type of user-specific information,
-despite efforts to do otherwise. As an example, consider OHTTP used for the purposes of hiding
+Partitioning alone is not a panacea for privacy. It is possible to apply partitioning in a way
```

Comment:
I wonder if we've used panacea too much at this point =)

Reply:
Heh, maybe =) I removed two of the three usages.